test/TestUtilities/Test.Utility/Signing/TrustedTestCert.cs

```diff
@@ -25,13 +24,19 @@ public class TrustedTestCert<T> : IDisposable
         public TrustedTestCert(T source,
             Func<T, X509Certificate2> getCert,
             StoreName storeName = StoreName.TrustedPeople,
-            StoreLocation storeLocation = StoreLocation.CurrentUser)
+            StoreLocation storeLocation = StoreLocation.CurrentUser,
+            TimeSpan? maximumValidityPeriod = null)
```

Comment:
I can implement this as an overload if you guys want, I just figured for test APIs default parameters is a bit easier to read via intellisense